src/Lesson12/AdditionalTask1.java

```diff
@@ -0,0 +1,32 @@
+package Lesson12;
+
+import java.io.*;
+
+//� �������� ����� ��������� �����, ������ ����� �� ����� �������. �����
+//������� ��������� ������ ������� ����, ������� ����� ��������� � ����
+//������ ����������
+public class AdditionalTask1 {
+    public static boolean palindrome(String word) {
+        String word1 = word.toLowerCase();
+        StringBuilder original = new StringBuilder(word1);
+        StringBuilder reverse = original.reverse();
+        return (reverse.toString()).equals(word1);
+    }
+
+    public static void main(String[] args) throws IOException {
+        FileReader fileReader = new FileReader("AddFile1.txt");
+        BufferedReader bufferedReader = new BufferedReader(fileReader);
+        FileWriter fileWriter = new FileWriter("Palindromefile1.txt");
+        BufferedWriter bufferedWriter = new BufferedWriter(fileWriter);
+        String result = bufferedReader.readLine();
```

Comment:
on .readLine() method I'm expecting batch of words (I mean all row, that contain a lot of words).